ios/FluentUI/Label/Label.swift

```diff
@@ -17,22 +17,22 @@ public enum TextColorStyle: Int, CaseIterable {
     case warning
     case disabled
 
-    func color(for window: UIWindow) -> UIColor {
+    func color(fluentTheme: FluentTheme) -> UIColor {
```

Comment:
interesting... you are actually tokenizing this control in this branch?

Reply:
Yeah this is the approach we've been mainly using to pull tokens for untokenized controls.